### PR TITLE
Add github actions for lpms

### PR DESCRIPTION
**Pull request by JamesWanglf**
We can switch from CircleCI to GH actions for CI and enable GPU tests.[#284]
The pre-requisites to enable GPU tests on GitHub actions would be like followings.
1. Install NVIDIA driver and Cuda toolkit on the host VM.
2. Install docker and nvidia-docker
3. Run [nvidia-patch](https://github.com/keylase/nvidia-patch)
    >In some cases, there is a restriction on maximum number of simultaneous NVENC video encoding sessions imposed by Nvidia to consumer-grade GPUs. To remove this restriction, I am using [NVENC patch](https://github.com/keylase/nvidia-patch).
4. Build docker image and run docker container
    >Run the following command to run docker container

    `nvidia-docker run -it --runtime=nvidia --gpus all --network host -e ORGANIZATION=livepeer -e REPOSITORY=lpms -e ACCESS_TOKEN=XXXXXXXXXX lpms-linux-runner:latest sh -c 'ldconfig -p | grep cuvid' 
	libnvcuvid.so.1 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libnvcuvid.so.1`
5. Push anything to GitHub repository to test

### CODE REVIEW

File: .github/runner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    && curl -O -L https://github.com/actions/runner/releases/download/v${RUNNER_VERSION}/actions-runner-linux-x64-${RUNNER_VERSION}.tar.gz \
+    && tar xzf ./actions-runner-linux-x64-${RUNNER_VERSION}.tar.gz && chown -R devops ~devops
+
+RUN LIBTENSORFLOW_VERSION=2.6.3 \
+  && curl -LO https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
+  && sudo tar -C /usr/local -xzf libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
+  && sudo ldconfig
+
+# Add mime type for ts
+RUN sudo echo '<?xml version="1.0" encoding="UTF-8"?><mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info"><mime-type type="video/mp2t"><comment>ts</comment><glob pattern="*.ts"/></mime-type></mime-info>'>>/usr/share/mime/packages/custom_mime_type.xml
```
Comment (cyberj0g) [inline .github/runner/Dockerfile:46]: Kudos for resolving this, it's been bugging multiple devs. I think it's worth opening corresponding issue in go-livepeer, to move the fix into `install_fffmpeg.sh` 👍

**Review by cyberj0g — APPROVED** [review]
Great work @JamesWanglf, LGTM! I think the next step is to create Helm chart for new test server to run on CoreWeave, @yondonfu @iameli please advice how best to do that.

**Review by oscar-davids — APPROVED** [review]
LGTM!

**Comment by yondonfu** [comment]
Created https://github.com/livepeer/livepeer-infra/issues/803 to track the work for creating + deploying the Helm chart that uses the Dockerfile in this PR for the self-hosted GH action workflow.

**Pull request merged**